<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageScriptingClass</class>
 <widget class="QWidget" name="PageScriptingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PageScripting</string>
  </property>
  <widget class="QTabWidget" name="fileTabs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>591</width>
     <height>381</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>127</width>
     <height>0</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::TabShape::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="ScriptEditor" name="mainEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <attribute name="title">
     <string>main</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QQuickWidget" name="qmlWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>420</width>
     <height>351</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>420</width>
     <height>0</height>
    </size>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>12</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>12</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>ScriptEditor</class>
   <extends>QWidget</extends>
   <header>scripteditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
